doc/design/communications.md

```diff
@@ -0,0 +1,92 @@
+# Communications
+
+This document discusses the communication channels that are possible between components, but not what flows over them. This is essentially Layer2/3 and Layer7 is discussed in the documents relating to specific components that talk over these channels.
+
+
+## ContainerVM to Appliance
+There are two types of communication that occur between a containerVM and the [VCH appliance](components.md#appliance):
+
+1. application level network traffic
+2. VIC control channel, which includes user interaction with the container console
+
+This section addresses the VIC control channel
+
+
+### Async - guestinfo
+The containerVM guestinfo is used as a persistent, asynch publishing mechanism for specific data related to container state, primarily:
+
+1. assigned DHCP address
+2. error reporting for [containerVM components](#components.md#container)
+3. exit status of the container process
+
+The other direction, appliance to containerVM, is addressed in a little more depth under [configuration](configuration.md#appliance)
```

Comment:
Confused about "the other direction" reference here. Other than what? Are you saying that the info you have described so far is just the stuff that the container is putting in there and that the VCH "sets" will be described in configuration.md?